app/static/app/js/components/NewTaskPanel.jsx

```diff
@@ -42,13 +43,13 @@ class NewTaskPanel extends React.Component {
   }
 
   componentDidMount(){
-    PluginsAPI.Dashboard.triggerAddNewTaskPanelItem({}, (item) => {
-        if (!item) return;
-
-        this.setState(update(this.state, {
-            items: {$push: [item]}
-        }));
-    });
+    // PluginsAPI.Dashboard.triggerAddNewTaskPanelItem({}, (item) => {
```

Comment:
Was this also affecting the plugin?

Reply:
Actually, just the import causes the problem. But since I commented out the import, I had to comment this too